src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -618,6 +618,7 @@ namespace
         {
             QComboBox* combobox = create_combobox("engine");
             combobox->setToolTip(m_params_metadata.get_path("lighting_engine.help"));
+            combobox->addItem("Bidirectional Path Tracer", "bdpt");
```

Comment:
I suggest making it the second item so that engines get sorted from the simplest (unidirectional PT) to the most advanced (probably VCM, eventually).